projects/admin/src/app/record/detail-view/document-detail-view/document-detail-view.component.html

```diff
@@ -26,122 +26,124 @@
+        <ul class="list-inline mb-0">
+          <li
+            class="list-inline-item"
+            *ngFor="let contribution of record.metadata.contribution; let last = last; let i = index"
+            [attr.id]="i | idAttribute:{prefix: 'doc-contribution'}"
+          >
+            <span *ngIf="!contribution.agent.pid" [innerHTML]="contribution | contributionFormat: true"></span>
+            <a
+            *ngIf="contribution.agent.pid"
+            [innerHTML]="contribution | contributionFormat: true"
+            [routerLink]="['/records', contributionTypeParam(contribution.agent), 'detail', contribution.agent.pid]">
+            </a>
+            {{ last ? '' : '; ' }}
+          </li>
+        </ul>
       </ng-container>
-    </ng-container>
 
-    <!-- CONTRIBUTION -->
-    <ng-container *ngIf="record.metadata.contribution && record.metadata.contribution.length > 0">
-      <ul class="list-inline mb-0">
-        <li
-          class="list-inline-item"
-          *ngFor="let contribution of record.metadata.contribution; let last = last; let i = index"
-          [attr.id]="i | idAttribute:{prefix: 'doc-contribution'}"
-        >
-          <span *ngIf="!contribution.agent.pid" [innerHTML]="contribution | contributionFormat: true"></span>
-          <a
-          *ngIf="contribution.agent.pid"
-          [innerHTML]="contribution | contributionFormat: true"
-          [routerLink]="['/records', contributionTypeParam(contribution.agent), 'detail', contribution.agent.pid]">
-          </a>
-          {{ last ? '' : '; ' }}
-        </li>
-      </ul>
-    </ng-container>
+      <!-- PUBLICATION STATEMENT -->
+      <ng-container *ngFor="let provision of provisionActivity | keyvalue">
+        <ng-container *ngIf="provision.key === 'bf:Publication'">
+          <ul class="list-unstyled mb-0">
+            <li *ngFor="let value of provision.value; let i = index"
+            [attr.id]="i | idAttribute:{prefix: 'doc-publication-statement'}">
+              {{ value }}
+            </li>
+          </ul>
+        </ng-container>
+      </ng-container>
 
-    <!-- PUBLICATION STATEMENT -->
-    <ng-container *ngFor="let provision of provisionActivity | keyvalue">
-      <ng-container *ngIf="provision.key === 'bf:Publication'">
+      <!-- EXTENT-->
+      <ng-container *ngIf="record.metadata.extent">
+        <div id="doc-extent">{{ record.metadata.extent }}</div>
+      </ng-container>
+
+      <!-- EDITION STATEMENT -->
+      <ng-container *ngIf="record.metadata.editionStatement && record.metadata.editionStatement.length > 0">
         <ul class="list-unstyled mb-0">
-          <li *ngFor="let value of provision.value; let i = index"
-          [attr.id]="i | idAttribute:{prefix: 'doc-publication-statement'}">
-            {{ value }}
+          <li *ngFor="let edition of getStatement(record.metadata.editionStatement); let i = index"
+          [attr.id]="i | idAttribute:{prefix: 'doc-edition-statement'}">
+          {{ edition.value }}
```

Comment:
here too (this seems cause a lot of problem to `git diff`)

Reply:
Done